src/sqltest/parts/notebook/model/cell.test.ts

```diff
@@ -185,6 +186,19 @@ suite('Cell Model', function (): void {
 	suite('Model Future handling', function (): void {
 		let future: TypeMoq.Mock<EmptyFuture>;
 		let cell: ICellModel;
+		const stdInDefaultMessage: nb.IStdinMessage = {
```

Comment:
😄 tests!

Reply:
I know! Wish I could have added tests over the angular component though